content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -13,127 +13,120 @@ aliases:
 keywords: [performance,scalability,scale,benchmarks]
 ---
 
-We follow a four-pronged approach to Istio performance characterization, tracking and improvements:
+Istio makes it easy to create a network of deployed services with rich routing,
+load balancing, service-to-service authentication, monitoring, and more - all
+without any changes to the application code. Istio strives to provide
+these benefits with minimal resource overhead and aims to support very
+large meshes with high request rates while adding minimal latency.
 
-* Code level micro-benchmarks
+The Istio data plane components, the Envoy proxies, handle data flowing through
+the system. The Istio control plane components, Pilot, Galley and Citadel, configure
+the data plane. The data plane and control plane have distinct performance concerns.
 
-* Synthetic end-to-end benchmarks across various scenarios
+## Performance summary for Istio {{< istio_release_name >}}
 
-* Realistic complex app end-to-end benchmarks across various settings
+The [Istio load tests](https://github.com/istio/tools/tree/master/perf/load) mesh consists
+of **1000** services and **2000** sidecars with `70k` mesh-wide requests per second (`rps`).
+After running the tests using Istio {{< istio_release_name >}}, we get the following results:
 
-* Automation to ensure performance doesn't regress
+- The Envoy proxy uses **0.6 vCPU** and **50 MB memory** per `1k rps` going through the proxy .
+- The `istio-telemetry` service uses **0.6 vCPU** per `1k` **mesh-wide** `rps`.
+- Pilot uses **1 vCPU** and `1.5 GB` of memory.
+- The Envoy proxy adds `10 ms` to the 99th percentile latency.
```

Comment:
@louiscryan argues that more folks look at 90th percentile latency in the Envoy community, and that we should be reporting at that level.